build.rs

```diff
@@ -92,6 +92,21 @@ fn main() -> Result<(), Box<dyn Error>> {
         println!("cargo:rerun-if-changed={}", file);
     }
 
+    // Print `cargo tree` for easier debugging if dependencies change.
+    let cargo_tree = Command::new(&cargo)
```

Comment:
We're kinda abusing the warning mechanism, and this adds to it. I suspect most builds don't care about the tree, so seeing that wall of text may be confusing. Can we dump the tree in a debug output file under OUT_DIR instead?

Reply:
AFAIK warnings from build scripts print always, not only on verbose builds.